src/components/component/component.spec.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Component from './component';
-import { StyleguideDeviceFrame } from '../../../index';
+import { StyleguideDeviceFrame } from '@/visual-helpers';
```

Comment:
This change should be in separate pull request (it is already a sixth new PR) with detailed description, what we are trying to achieve here.

I believe I recall what it can be, but let's move this in separate scope.

Main reason - scope of pull-request - let's keep it on feature based approach as we discussed before:)

I recommend for now all paths to keep as is and fix it in separate PR for the whole project.